tests/unit/conftest.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
+
+import pytest
+
+from entitysdk.client import Client
+from entitysdk.common import ProjectContext
+
+
+@pytest.fixture
+def project_context():
+    return ProjectContext(
+        project_id="103d7868-147e-4f07-af0d-71d8568f575c",
+        virtual_lab_id="103d7868-147e-4f07-af0d-71d8568f575c",
+    )
+
+
+@pytest.fixture
+def client(project_context):
+    with patch("entitysdk.client.httpx"):
```

Comment:
Instead of patching, we can add an optional http_client parameter to the Client and inject it as a dependency.

Reply:
Indeed. Done.